lib/landing_session.js

```diff
@@ -14,6 +15,33 @@ class LandingSession extends Session {
     super(dir, prid, config);
     this.cli = cli;
     this.req = req;
+    const { upstream } = this;
+    const upstreamHref = runSync('git', ['config', '--get',
+      `remote.${upstream}.url`]);
+    let warned = false;
+    try {
+      const { hostname, pathname } = new URL(upstreamHref);
+      if (hostname !== 'github.com') {
+        warned = true;
+        cli.warn('Remote repository URL does not point to the expected ' +
+          'host "github.com"');
+      }
+      if (/^\/nodejs\/node(?:\.git)?$/.test(pathname) !== true) {
```

Comment:
Can you change this to `new RegExp(`^\/${this.owner}\/${this.repo}(?:\.git)?$/`)` so these can be configured? Also the warning below (`this.owner` and `this.repo` come from the `Session` class this is inheriting, they default to `nodejs` and `node`).